soccer/gameplay/skills/move_direct.py

```diff
@@ -42,6 +43,12 @@ def threshold(self):
     def threshold(self, value):
         self._threshold = value
 
+    def velocity_check(self):
+        if (self.check_velocity):
+            return self.robot.vel.mag() < .05
```

Comment:
agreed, that should be a bigger issue. We'll target to have it fixed before or during fall 2017 then.